### DISCUSSION TITLE
Garak Probes would they in the future able to mitigate the below risks

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [NVIDIA/garak#915] [Q&A] [answered]
Hi,

Wanted to know if would be technically feasible for Garak to mitigate the below risks .(Even in the future)

1) Child pornography present in bad llm model [refer](https://www.washingtonpost.com/technology/2023/12/20/ai-child-pornography-abuse-photos-laion/) 

2) LLM model containing 3rd party copywritten information or intellectual property 

3) LLMs with backdoors to steel proprietary information from company 

4) LLM responding with factually wrong response 

5) LLM model being biased

**Comment by leondz** [comment, marked as answer]
Thanks for the question
1. Image scanning is out of scope for now - we support multimodal input but don't get anything beyond text and, where available, model files for failures
2. Some probes already check for this, e.g. `leakreplay`
3. Models can't do this in and of themselves - it depends on the rest of the stack, which is out of scope
4. We have probes for this also, e.g. `misleading`, `snowball` and `packagehallucination`
5. Bias is out of scope for garak
